filtering/flink_table_api_python/README.md

```diff
@@ -0,0 +1,146 @@
+<!-- title: How to filter Kafka messages in Python using Flink's Table API for Confluent Cloud -->
+<!-- description: In this tutorial, learn how to filter Kafka messages in Python using Flink's Table API for Confluent Cloud, with step-by-step instructions and supporting code. -->
+
+# How to filter Kafka messages in Python using Flink's Table API for Confluent Cloud
+
+## Prerequisites
+
+* A [Confluent Cloud](https://confluent.cloud/signup) account
+* The [Confluent CLI](https://docs.confluent.io/confluent-cli/current/install.html) installed on your machine
+* [Python](https://www.python.org/downloads/) 3.8 or later
+* Java 21, e.g., follow the OpenJDK installation instructions [here](https://openjdk.org/install/) if you don't have Java. Validate that `java -version` shows version 21. _Note: Java is required since the Flink Python API uses [Py4J](https://www.py4j.org/) to communicate with the JVM under the hood._
+* Clone the `confluentinc/tutorials` GitHub repository (if you haven't already) and navigate to the `tutorials` directory:
+    ```shell
+    git clone git@github.com:confluentinc/tutorials.git
+    cd tutorials
+    ```
+
+## Provision Confluent Cloud infrastructure
+
+If you already have the Confluent Cloud resources required to populate a Table API client configuration file, e.g., from running a different tutorial, you may skip to the [next step](#inspect-the-code) after creating or copying the properties file as documented [here](https://docs.confluent.io/cloud/current/flink/reference/table-api.html#properties-file) to `filtering/flink_table_api_python/cloud.properties` within the top-level `tutorials` directory.
+
+If you need to create the Confluent Cloud infrastructure needed to run this tutorial, the `confluent-flink-quickstart` CLI plugin creates the resources that you need to get started with Confluent Cloud for Apache Flink. Install it by running:
+
+```shell
+confluent plugin install confluent-flink-quickstart
+```
+
+Run the Flink quick start plugin as follows to create the Confluent Cloud resources needed for this tutorial and generate a Table API client configuration file. Note that you may specify a different cloud provider (`gcp` or `azure`) or region. You can find supported regions in a given cloud provider by running `confluent flink region list --cloud <CLOUD>`.
```

Comment:
shouldn't quick start be - quickstart?

Reply:
I try to use "quick start" even though either one is probably acceptable. just because in docs and CD we avoid "quickstart". But the plugin has `quickstart` in it so it's a little confusing - i skirted the issue by removing it and just saying "Run the plugin as follows ..."